doc/source/skips/2-values.rst

```diff
@@ -0,0 +1,178 @@
+
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Author: Josh Warner
+:Author: François Boulogne
+:Author: Emmanuelle Gouillart
+:Author: Mark Harfouche
+:Author: Lars Grüter
+:Author: Egor Panfilov
+:Author: Gregory Lee
+:Status: Active
+:Type: Process
+:Created: 2018-12-08
+:Resolved:
+:Resolution:
+:Version effective: 0.16
+
+Abstract
+--------
+
+scikit-image should adopt the document below as its mission statement. This
+statement will feature prominently in the scikit-image home page and readme,
+as well as the contributor and core developer guides. Decisions about the API
+and the future of the library would be referenced against this document. (See
+:ref:`governance`.)
+
+In July 2018, I published a blog post that broadly outlined what I would want
+from a roadmap for scikit-image [1]_, but requested comments from the community
+before it would be finalized. I consider that we have collected comments for
+long enough and can move forward with formal adoption. Most comments were
+positive, so below I'll just summarize the “negative” comments under “rejected
+ideas”.
+
+Detailed description
+--------------------
+
+(Or: What problem does this proposal solve?)
+
+Over the past few years, scikit-image has been slightly “adrift”, with new and
+old contributors coming in, adding what small bits they need at the time, and
+disappearing again for a while. This is *fine* and we want to encourage more of
+it, but it also lacks direction. Additionally, without a concerted roadmap to
+concentrate effort, many of these contributions just fall by the wayside, as it
+is difficult for new contributors to live up to our stringent (and largely
+unwritten) standards of code.
+
+Implementation
+--------------
+
+Our mission
+***********
+
+scikit-image aims to be the essential toolkit for scientific image analysis in
+Python. We accomplish this by:
+
+- being **easy to use and install**. We are careful in taking on new
+  dependencies, and sometimes cull existing ones, or make them optional. All
+  functions in our API have thorough docstrings clarifying expected inputs and
+  outputs.
+- providing a **consistent API**. Conceptually identical arguments have the
+  same name and position in a function signature.
+- **ensuring accuracy**. Test coverage is close to 100% and code is reviewed by
+  at least two core developers before being included in the library.
+- **caring for users’ data**. We have a functional [2]_ API and don't modify
+  input arrays unless explicitly directed to do so.
+- promoting **education in image processing** with extensive pedagogical
+  documentation.
+
+Our values
+**********
+
+- We are inclusive. We continue to welcome and mentor newcomers who are
+  making their first contribution.
+- We are community-driven. Decisions about the API and features are driven by
+  our users' requirements, not by the whims of the core team. (See
+  :ref:`governance`.)
```

Comment:
Note that our governance document *does not make it explicit* how the general community can contribute to decision-making.

Reply:
Hmm, a very good point. I'm heading to bed so I'll have to fix this tomorrow/day after.

Reply:
the community can take part and shape the discussion, but they cannot vote or make decisions on the project.

Reply:
Added a section in the governance doc!

Reply:
Note: the contributing doc needs improvement on this front, but I don't think it should hold up this PR.